### ISSUE TITLE
chord definitions for 13, etc., need editing

### ISSUE THREAD

**Issue by chrisfargen** [paulrosen/abcjs#559]
Example:

```
K:C
"G13" z8 |
```

I looked at the definition and saw this:

```
'13': [ 0, 4, 7, 10, 14, 18 ],
```

But it should be something like this:

```
'13': [ 0, 4, 7, 10, 14, 21 ],
```

There are others with similar issues, but I haven't made an exhaustive list yet.

**Comment by paulrosen** [comment]
Ok, let me know what you find.

**Comment by paulrosen** [comment]
I went through them and think I found a number. Here is my new proposed list that will be in the next release. Let me know if that looks right to you:
```
		// diminished (all flat 5 chords)
		'dim': [ 0, 3, 6 ],
		'°': [ 0, 3, 6 ],
		'˚': [ 0, 3, 6 ],

		'dim7': [ 0, 3, 6, 9 ],
		'°7': [ 0, 3, 6, 9 ],
		'˚7': [ 0, 3, 6, 9 ],

		'ø7': [ 0, 3, 6, 10 ],
		'm7(b5)': [ 0, 3, 6, 10 ],
		'm7b5': [ 0, 3, 6, 10 ],
		'm7♭5': [ 0, 3, 6, 10 ],
		'-7(b5)': [ 0, 3, 6, 10 ],
		'-7b5': [ 0, 3, 6, 10 ],

		'7b5': [ 0, 4, 6, 10 ],
		'7(b5)': [ 0, 4, 6, 10 ],
		'7♭5': [ 0, 4, 6, 10 ],

		'7(b9,b5)': [ 0, 4, 6, 10, 13 ],
		'7b9,b5': [ 0, 4, 6, 10, 13 ],
		'7(#9,b5)': [ 0, 4, 6, 10, 15 ],
		'7#9b5': [ 0, 4, 6, 10, 15 ],
		'maj7(b5)': [ 0, 3, 6, 11 ],
		'maj7b5': [ 0, 3, 6, 11 ],
		'13(b5)': [ 0, 4, 6, 10, 14, 18 ],
		'13b5': [ 0, 4, 6, 10, 14, 18 ],

		// minor (all normal 5, minor 3 chords)
		'm': [ 0, 3, 7 ],
		'-': [ 0, 3, 7 ],
		'm6': [ 0, 3, 7, 9 ],
		'-6': [ 0, 3, 7, 9 ],
		'm7': [ 0, 3, 7, 10 ],
		'-7': [ 0, 3, 7, 10 ],

		'-(b6)': [ 0, 3, 7, 8 ],
		'-b6': [ 0, 3, 7, 8 ],
		'-6/9': [ 0, 3, 7, 9, 14 ],
		'-7(b9)': [ 0, 3, 7, 10, 13 ],
		'-7b9': [ 0, 3, 7, 10, 13 ],
		'-maj7': [ 0, 3, 7, 11 ],
		'-9+7': [ 0, 3, 7, 11, 13 ],
		'-11': [  0, 3, 7, 11, 14, 16 ],

		// major (all normal 5, major 3 chords)
		'M': [ 0, 4, 7 ],
		'6': [ 0, 4, 7, 9 ],
		'6/9': [ 0, 4, 7, 9, 14 ],

		'7': [ 0, 4, 7, 10 ],
		'9': [ 0, 4, 7, 10, 14 ],
		'11': [ 0, 4, 7, 10, 14, 17 ],
		'13': [ 0, 4, 7, 10, 14, 21 ],
		'7b9': [ 0, 4, 7, 10, 13 ],
		'7♭9': [ 0, 4, 7, 10, 13 ],
		'7(b9)': [ 0, 4, 7, 10, 13 ],
		'7(#9)': [ 0, 4, 7, 10, 15 ],
		'7#9': [ 0, 4, 7, 10, 15 ],
		'(13)': [ 0, 4, 7, 10, 14, 21 ],
		'7(9,13)': [ 0, 4, 7, 10, 14, 21 ],
		'7(#9,b13)': [ 0, 4, 7, 10, 15, 20 ],
		'7(#11)': [ 0, 4, 7, 10, 14, 18 ],
		'7#11': [ 0, 4, 7, 10, 14, 18 ],
		'7(b13)': [ 0, 4, 7, 10, 20 ],
		'7b13': [ 0, 4, 7, 10, 20 ],
		'9(#11)': [ 0, 4, 7, 10, 14, 18 ],
		'9#11': [ 0, 4, 7, 10, 14, 18 ],
		'13(#11)': [ 0, 4, 7, 10, 18, 21 ],
		'13#11': [ 0, 4, 7, 10, 18, 21 ],

		'maj7': [ 0, 4, 7, 11 ],
		'∆7': [ 0, 4, 7, 11 ],
		'Δ7': [ 0, 4, 7, 11 ],
		'maj9': [ 0, 4, 7, 11, 14 ],
		'maj7(9)': [ 0, 4, 7, 11, 14 ],
		'maj7(11)': [ 0, 4, 7, 11, 16 ],
		'maj7(#11)': [ 0, 4, 7, 11, 18 ],
		'maj7(13)': [ 0, 4, 7, 11, 21 ],
		'maj7(9,13)': [ 0, 4, 7, 11, 14, 21 ],

		'7sus4': [ 0, 5, 7, 10 ],
		'm7sus4': [ 0, 5, 7, 10 ],
		'sus4': [ 0, 5, 7 ],
		'sus2': [ 0, 2, 7 ],
		'7sus2': [ 0, 2, 7, 10 ],
		'9sus4': [ 0, 5, 7, 14 ],
		'13sus4': [ 0, 5, 7, 21 ],

		// augmented (all sharp 5 chords)
		'aug7': [ 0, 4, 8, 10 ],
		'+7': [ 0, 4, 8, 10 ],
		'+': [ 0, 4, 8 ],
		'7#5': [ 0, 4, 8, 10 ],
		'7♯5': [ 0, 4, 8, 10 ],
		'7+5': [ 0, 4, 8, 10 ],
		'9#5': [ 0, 4, 8, 10, 14 ],
		'9♯5': [ 0, 4, 8, 10, 14 ],
		'9+5': [ 0, 4, 8, 10, 14 ],
		'-7(#5)': [ 0, 3, 8, 10 ],
		'-7#5': [ 0, 3, 8, 10 ],
		'7(#5)': [ 0, 4, 8, 10 ],
		'7(b9,#5)': [ 0, 4, 8, 10, 13 ],
		'7b9#5': [ 0, 4, 8, 10, 13 ],
		'maj7(#5)': [ 0, 4, 8, 11 ],
		'maj7#5': [ 0, 4, 8, 11 ],
		'maj7(#5,#11)': [ 0, 4, 8, 11, 18 ],
		'maj7#5#11': [ 0, 4, 8, 11, 18 ],
		'9(#5)': [ 0, 4, 8, 10, 14 ],
		'13(#5)': [ 0, 4, 8, 10, 14, 21 ],
		'13#5': [ 0, 4, 8, 10, 14, 21 ]
```

**Comment by chrisfargen** [comment]
(1.) These:

		'maj7(b5)': [ 0, 3, 6, 11 ],
		'maj7b5': [ 0, 3, 6, 11 ],

should be:

		'maj7(b5)': [ 0, 4, 6, 11 ],
		'maj7b5': [ 0, 4, 6, 11 ],

----

(2.) These:

		'13(b5)': [ 0, 4, 6, 10, 14, 18 ],
		'13b5': [ 0, 4, 6, 10, 14, 18 ],

should be:

		'13(b5)': [ 0, 4, 6, 10, 14, 21 ],
		'13b5': [ 0, 4, 6, 10, 14, 21 ],

----

(3.) Regarding this:

		'-9+7': [ 0, 3, 7, 11, 13 ],

I'm not saying it's not a thing, but I can't find any info about that chord symbol. However, there is another chord that is similar but less exotic that does not appear on the current list.

		'-maj9': [ 0, 3, 7, 11, 14 ],
		'-∆9': [ 0, 3, 7, 11, 14 ],
		'mM9': [ 0, 3, 7, 11, 14 ],

----

(4.) Below this:

		'6/9': [ 0, 4, 7, 9, 14 ],


You might add these:

		'6add9': [ 0, 4, 7, 9, 14 ],
		'69': [ 0, 4, 7, 9, 14 ],

----

I haven't made it through the major and augmented chords yet.

**Comment by musikboden** [comment]
I think the 
maj7(#11)': [ 0, 4, 7, 11, 18 ]
should be [0, 4, 6, 11] (often voiced like this in jazz harmonies) or [0, 4, 11, 18]
It's more common to not have the fifth in a #11 chord, I would say this is the same chord as maj7(b5) (which I would consider wrongly written cause you would normally improvise on both the fifth and the sharp fourth.
But that's only my opinion

**Comment by paulrosen** [comment]
I have tried to find some definitive guide for leaving out the 5th in 11th and 13th chords or leaving out the 11th in 13th chords. I do see guides say "it's common" like you did. I almost removed them today. I'd be happy to. I think the chords sound pretty dense.

I'm not a particular good judge though. I don't listen much to music that uses those chords so I don't have a feel for it.

**Comment by musikboden** [comment]
Yes, I agree they sound dense, I think you should leave out the 5th, at least in all chords that has a sharp 11
But this is only my opinion.

The scheme for the chords, are they played the way they are structured?


This
'-9+7': [ 0, 3, 7, 11, 13 ],
looks odd, for me -9 is a minor chord including 7 and 9 but the +7 implies that it should include a maj7 as well, never seen this before

'-11': [  0, 3, 7, 11, 14, 16 ],
should be [0, 3, 7, 10, 14, 17 ]

This
'11': [ 0, 4, 7, 10, 14, 17 ],
should be 
[ 0, 7, 10, 14, 17 ] I took away the major third because of the dissonance between the major third and the 11, the chord is more of an unreleased chord, like sus4 with extensions

This
'maj7(11)': [ 0, 4, 7, 11, 16 ],
I've never seen this chord and "16" is the same tone as "4", major third

This
'maj7(13)': [ 0, 4, 7, 11, 21 ],
should be
[ 0, 4, 7, 11, 14, 21 ]

This
'm7sus4': [ 0, 5, 7, 10 ],
is a bit strange, the "sus" implies that the fourth is suspended and on it's way to be released to the third, I think it should be spelled m7add4 or m7add11, and contain these tones:
[0, 3, 5, 7 ]

This
'9sus4': [ 0, 5, 7, 14 ],
should also include the 7
[ 0, 5, 7, 10, 14 ]

and this
'13sus4': [ 0, 5, 7, 21 ],
should include the 7 and 9
[ 0, 5 ,7, 10, 14, 21 ]

**Comment by paulrosen** [comment]
I don't remember where all these chords came from - I definitely looked at a number of charts to get the names. It was probably from a real book.

I agree that some of the chords are odd.

**Comment by paulrosen** [comment]
Fixed in 6.0.0-beta.22
